packages/platform/atoms/hooks/connect/useCheck.ts

```diff
@@ -1,20 +1,30 @@
 import { useQuery, useQueryClient } from "@tanstack/react-query";
 
 import type { CALENDARS } from "@calcom/platform-constants";
-import { SUCCESS_STATUS, ERROR_STATUS } from "@calcom/platform-constants";
-import type { ApiResponse, ApiErrorResponse } from "@calcom/platform-types";
+import { ERROR_STATUS, SUCCESS_STATUS } from "@calcom/platform-constants";
+import type { ApiErrorResponse, ApiResponse } from "@calcom/platform-types";
 
 import http from "../../lib/http";
 import { useAtomsContext } from "../useAtomsContext";
 
 export interface UseCheckProps {
   onCheckError?: OnCheckErrorType;
   calendar: (typeof CALENDARS)[number];
+  config?: Partial<{
+    initialData: {
+      status: string;
```

Comment:
The status can be specified further - `SUCCESS_STATUS | ERROR_STATUS`

Reply:
Just implemented this as well. Nice!!